go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/husniadil/uuid
 go 1.15
 
 require (
-	github.com/google/uuid v1.2.0
+	github.com/google/uuid v1.3.0
```

Comment:
The update from `github.com/google/uuid v1.2.0` to `v1.3.0` introduces a new optional randomness pool mode for V4 UUID generation. While this can improve performance, it is important to assess whether this feature aligns with your application's security requirements. If your application is security-sensitive, consider avoiding the use of this feature or explicitly disabling it.